halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Features.java

```diff
@@ -49,6 +49,12 @@ public NodeIterator getChildren() {
           "Artifacts are not configurable prior to this release. Will be stable at a later release.")
   private Boolean artifacts;
 
+  @ValidForSpinnakerVersion(
+      lowerBound = "1.5.0",
```

Comment:
I could be off here but this seems a little low, the rewrite was available starting in 1.5?

Reply:
👍